bootstrap

```diff
@@ -20,19 +20,31 @@ main(_) ->
     Deps = get_deps(),
     [fetch_and_compile(Dep, Deps) || Dep <- BaseDeps],
 
+    %% To check whether a build was triggered by cron, examine the TRAVIS_EVENT_TYPE
+    %% environment variable to see if it has the value cron.
+    %% https://docs.travis-ci.com/user/cron-jobs/
+    Profile = case os:getenv("TRAVIS_EVENT_TYPE") of
+                    "cron" ->
+                        io:format("nightly build"),
+                        "nightly";
+                    _ -> "bootstrap"
+              end,
+
     %% Build rebar3 modules with compile:file
-    bootstrap_rebar3(),
+    bootstrap_rebar3(Profile),
```

Comment:
that does not seem to be a correct usage of profiles. I.e. the actual profile could be `bootstrap+nightly` -- this appears to be about the output path, but I'm not too sure what the intent is here. Why does a nightly build require different settings?